core/authenticate/service.go

```diff
@@ -251,6 +293,23 @@ func (s Service) FinishFlow(ctx context.Context, request RegistrationFinishReque
 		return response, nil
 	}
 
+	if request.Method == PassKeyAuthMethod.String() {
+		if request.StateConfig["type"] == "register" {
```

Comment:
what if instead of asking user to send the `type`, we store it in flow metadata when the flow is created? Because we anyway have the flow id in `state` variable, we can check if its register or login? So from frontend, we don't have to worry about sending if its register/login.

Reply:
added this in the metadata of flow